styles/style.css

```diff
@@ -1,13 +1,14 @@
-* {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
+*, *:before, *:after {
+  box-sizing: inherit;

```

Comment:
! wazne 

domyslnie bedzie pobierane z rodzica - dojdzie do html - ustawi border-box. 

ale jesli uzywamy kodu zewnetrznego - np widget chatu, albo popup newslettera na zasadzie copy-paste, a on byl pisany pod content-box to nie rozpadna nam sie ich style dzieki temu inherit - w jednym miejscu ustawimy ze `.widget-chat { box-sizing: content-box; }` i wtedy caly widget sie naprawi sam. 


blokowanie marginesow i paddingow wszystkim elementom to slaby pomysl - zepsuja sie listy, naglowki, tabele